### PR TITLE
Added swagger docs

### PR DESCRIPTION
**Pull request by agustina-mazzeo**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -30,6 +31,9 @@
     "passport": "^0.6.0",
     "passport-jwt": "^4.0.1",
     "passport-local": "^1.0.0",
+    "swagger-autogen": "^2.23.1",
```
Comment (fedecote) [inline backend/package.json:34]: remove unused package

File: backend/src/api/user/user.controller.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ class UserController {
   constructor(private userReadService: IUserReadService, private accountReadService: IAccountReadService) {}
   public getUsers = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
     try {
-      res.status(200).json(await this.userReadService.getAll());
+      res.status(200).json({ data: await this.userReadService.getAll() });
```
Comment (fedecote) [inline backend/src/api/user/user.controller.ts:12]: why did you change this?

Comment (agustina-mazzeo) [reply backend/src/api/user/user.controller.ts:12]: So that it matches the response object of all the other endpoints.
